.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: go
 
 go:
-  - "1.9"
-  - "1.10"
-  - "1.11"
+  - "1.9.x"
+  - "1.10.x"
```

Comment:
This bumps the 1.10 patch version from `go1.10 linux/amd64` to `go1.10.4 linux/amd64`.

1.10 is implicitly 1.10.0 -- try `eval "$(gimme "1.10.0")"` to see that there is no 1.10.0 version.